hieroplatform/src/main/java/org/hiero/sketch/table/RecordOrder.java

```diff
@@ -179,4 +180,85 @@ public IndexComparator getComparator(@NonNull final Table table) {
         }
         return mergeLeft;
     }
+
+    /**
+     * Given two Tables in sorted order, decide the order in which to merge them. If two rows
+     * are equal, they will be combined to a single row (possibly with a larger  count).
+     * @param left The left side Table
+     * @param right the right side Table
+     * @return A Integer array whose entries encode where the next element in the sorted order comes
+     * from. Its i^th element is -1 if the i^th element in merged table
+     * comes form the Left, 1 if it comes from the Right, 0 if the two are equal.
+     */
+    public List<Integer> getIntMergeOrder(@NonNull final Table left, @NonNull final Table right) {
+        if (!left.schema.equals(right.schema)) {
+            throw new RuntimeException("Tables do not have matching schemas");
+        }
+        final int  leftLength = left.getNumOfRows();
+        final int  rightLength = right.getNumOfRows();
+        final List<Integer> merge = new ArrayList<>();
+        int i = 0, j = 0;
+        int outcome;
+        while ((i < leftLength) && (j < rightLength)) {
+            outcome = 0;
+            for (final ColumnSortOrientation ordCol : this.sortOrientationList) {
```

Comment:
The previous implementation could probably have also used this version which gives more information that a Boolean array (but Quantiles does not need it).